File: problems/prob05/prob5.py
from random import randrange


def generateList(num):
    array = []
    if num > 40:
        for x in range(40, num + 1):
            if x % 2 != 0:
                array.append(x)
    else:
        for x in range(0, 40 + 1):
            if x % 2 != 0:
                array.append(x)
    return array

rand_num = randrange(100)
print("The number is {0}".format(rand_num))
print(generateList(rand_num))

File: problems/prob05/prob5.js
function generateList(randNum) {
  list = [];
  if (randNum > 40) {
    for (var i = 40; i <= randNum; i++) {
      if (i % 2 !== 0){
        list.push(i);
      }
    }
  } else {
    for (var j = 0; j < 40; j++) {
      if (j % 2 !== 0) {
        list.push(j);
      }
    }
  }
  return list;
}

var randNum = (Math.floor((Math.random() * 100)));
console.log('The number is ' + randNum);
console.log(generateList(randNum));